lib/util/util.PrefixingMessageProvider.js

```diff
@@ -0,0 +1,53 @@
+this.util = this.util || {};
+
+util.PrefixingMessageProvider = ( function() {
+	'use strict';
+
+	/**
+	 * Calls another message provider with prefixed message keys
```

Comment:
Being picky: Can you write \`MessageProvider\` in class- and function-level documentation, please?